lib/logstash/outputs/elasticsearch/common.rb

```diff
@@ -114,6 +114,15 @@ def maximum_seen_major_version
       client.maximum_seen_major_version
     end
 
+
+    def routing_field_name
+      maximum_seen_major_version >= 6 ? :routing : :_routing
```

Comment:
since we'll use the non underscored version in 6.x and 7.x here, can we confirm that all versions of 6.x support `routing` too?